Services/Component/classes/class.ilObjComponentSettingsGUI.php

```diff
@@ -492,11 +477,11 @@ function activatePlugin()
 		{
 			ilUtil::sendFailure($e->getMessage, true);
 		}
-			
+
 		$ilCtrl->setParameter($this, "ctype", $_GET["ctype"]);
 		$ilCtrl->setParameter($this, "cname", $_GET["cname"]);
-		$ilCtrl->setParameter($this, "slot_id", $_GET["slot_id"]);		
-		
+		$ilCtrl->setParameter($this, "slot_id", $_GET["slot_id"]);
```

Comment:
Isn't the re a saveParameter for exactly this usecase here?

`$ilCtrl->setParameter($this, "ctype", $_GET["ctype"]); == $ilCtrl->saveParameter($this, "ctype");`